contracts/JBController.sol

```diff
@@ -1047,4 +1047,17 @@ contract JBController is IJBController, JBOperatable {
         JBConstants.MAX_RESERVED_RATE - _reservedRate
       ) - _unprocessedTokenBalanceOf;
   }
+
+  /**
+    @dev See {IERC165-supportsInterface}.
+  */
+  function supportsInterface(bytes4 interfaceId)
```

Comment:
since this is a public fn, move it up in the file to the end of the External transactions section.

Reply:
[Moved it](https://github.com/jbx-protocol/juice-contracts-v2/blob/8979c45bd1b8997eacc09969424c392eb864359d/contracts/JBController.sol#L311)